usaspending_api/common/helpers/endpoint_documentation.py

```diff
@@ -115,6 +118,7 @@ def validate_docs(url, url_object, master_endpoint_list):
         if re.fullmatch(pattern, endpoint):
             break
     else:
-        messages.append("No URL found in {} that matches {} ({})".format(ENDPOINTS_MD, url, qualified_name))
+        if url not in _EXCLUDED_URLS:
+            messages.append("No URL found in {} that matches {} ({})".format(ENDPOINTS_MD, url, qualified_name))
```

Comment:
Nice idea and work in this file.

Reply:
Thank you, I think it establishes a way forward for future needs too. Like we could have used this when we wanted to remove the agency office autocomplete.